application/src/Controller/ApiController.php

```diff
@@ -243,8 +242,8 @@ protected function getErrorResult(MvcEvent $event, \Exception $error)
      *
      * @param string
      * @return mixed
-     * @throws Exception\DomainException if no JSON decoding functionality is
-     *     available.
+     * @throws Exception\InvalidJsonException if no JSON decoding functionality
```

Comment:
The note about this specific exception actually probably dates to when we used Zend's JSON component here. We should update this `@throws` but need to fix the actual comment as well, since it actually throws on JSON decoding errors or if the content is a scalar.

Reply:
Ok. Fixed in next commit.